waku/waku_store/rpc.nim

```diff
@@ -156,7 +158,7 @@ proc toAPI*(rpc: HistoryQueryRPC): HistoryQuery =
     pageSize = if rpc.pagingInfo.isNone() or rpc.pagingInfo.get().pageSize.isNone(): 0'u64
                else: rpc.pagingInfo.get().pageSize.get()
 
-    ascending = if rpc.pagingInfo.isNone() or rpc.pagingInfo.get().direction.isNone(): true
+    ascending = if rpc.pagingInfo.isNone() or rpc.pagingInfo.get().direction.isNone(): HistoryQueryAscendingDefaultValue
```

Comment:
Would it be possible to have the changes in `rpc.nim` in a separate PR? I think that will give better scope to the change. Even though this seems a simple change, I think is convenient to separate it.

On the other hand, I'd change the attribute's name from `ascending` to `direction`.

Reply:
Moved to https://github.com/waku-org/nwaku/pull/2263.